src/opm/output/eclipse/Summary.cpp

```diff
@@ -775,6 +775,7 @@ inline std::vector< const Well* > find_wells( const Schedule& schedule,
 
     if( type == ECL_SMSPEC_GROUP_VAR ) {
         if( !schedule.hasGroup( name ) ) return {};
+        if (timestep!=0) timestep--;
 
```

Comment:
What about adding a variable for this. i.e.  previous_timestep_index = timestep - 1 to avoid misuse later.